tools/cockpit.spec

```diff
@@ -527,6 +536,11 @@ test -f %{_bindir}/firewall-cmd && firewall-cmd --reload --quiet || true
 %systemd_postun_with_restart cockpit.socket
 %systemd_postun_with_restart cockpit.service
 
+%if 0%{?suse_version}
+%verifyscript ws
+%verify_permissions -e %{_libexecdir}/cockpit-session
```

Comment:
I'm also curious what this thing does?

Reply:
Since rpm can't verify the permissions (which makes me realize the patch was incomplete) there has to be a verifyscript that checks if file permissions match the config.